etc/kayobe/kolla/config/bifrost.yml

```diff
@@ -0,0 +1,8 @@
+# yamllint disable-file
+---
+# Use prebuilt release train images from Ark.
+{% if stackhpc_download_overcloud_host_images | bool %}
+overcloud_dib_build_host_images: false
```

Comment:
This is a Kayobe variable and doesn't apply here.

Reply:
You could override the default in `etc/kayobe/overcloud-dib.yml`, setting it to

```
"{{ os_distribution == 'rocky' and not stackhpc_download_overcloud_host_images | bool }}"
```

Reply:
Done